src/lib/html.ts

```diff
@@ -101,7 +101,12 @@ function create_impl_reports(data: ReportData): string {
             add_child(header_row, 'th', 'Id');
             add_child(header_row, 'th', 'Req');
             for (const impl of (consolidated ? data.consolidated_implementers : data.implementers)) {
-                const head = impl.variant !== 'consolidated' ? `${impl.name}<br />${impl.variant}` : impl.name;
+                let head: string;
+                if (impl.variant !== undefined) {
+                    head = impl.variant !== 'consolidated' ? `${impl.name}<br />${impl.variant}` : impl.name;
```

Comment:
I'm having trouble parsing this (especially telling how it's different from the old version) — could use a comment.

Reply:
The variant field in the implementation report is optional, and the original code did not take that into account. As a result, in some cases, an extra "(undefined)" field appeared in the output. Bug...